components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2Authenticator.java

```diff
@@ -299,40 +472,43 @@ protected Map<ClaimMapping, String> getSubjectAttributes(OAuthClientResponse tok
 
         Map<ClaimMapping, String> claims = new HashMap<>();
 
-        try {
-            String accessToken = token.getParam(OIDCAuthenticatorConstants.ACCESS_TOKEN);
-            String url = getUserInfoEndpoint(token, authenticatorProperties);
-            String json = sendRequest(url, accessToken);
-
-            if (StringUtils.isBlank(json)) {
-                if(log.isDebugEnabled()) {
-                    log.debug("Empty JSON response from user info endpoint. Unable to fetch user claims." +
-                            " Proceeding without user claims");
-                }
-                return claims;
-            }
+        if (token != null) {
```

Comment:
Can we consider the inverse of this?

Reply:
Fixed f2aa0c1666f9b5234e7d113125fadd55008081cd